test/min_items_test.rb

```diff
@@ -11,6 +11,6 @@ def test_minitems_nils
     errors = JSON::Validator.fully_validate(schema, [nil])
     assert_equal(errors.length, 1)
     assert(errors[0] !~ /minimum/)
-    assert(errors[0] =~ /NilClass/)
+    assert(errors[0] =~ /null/)
```

Comment:
Do you know why this has changed? (I assume something related to nil in ruby 2.4?)